components/Templates/class-pods_templates.php

```diff
@@ -221,6 +221,15 @@ public function add_metaboxes( $slug, $post = false ) {
 				'groups' => array(),
 			)
 		);
+		add_meta_box(
+			'pod_reference', __( 'Magic Tag Reference', 'pods' ), array(
+				$this,
+				'render_metaboxes_custom'
+			), '_pods_template', 'side', 'default', array(
+				'slug' => 'tag_reference',
+				'groups' => array()
```

Comment:
* :no_entry_sign: Each line in an array declaration must end in a comma
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 4ff5747177efdf40a6f983794bc3282a</sub>